models/src/anemoi/models/layers/attention.py

```diff
@@ -252,7 +252,9 @@ def forward(
         if window_size is not None and (self.mask is None or tuple(self.mask.shape) != (sequence_len, sequence_len)):
             self.update_mask(sequence_len, window_size=window_size, device=query.device)
 
-        with torch.nn.attention.sdpa_kernel(backends=[torch.nn.attention.SDPBackend.MATH]):
+        with torch.nn.attention.sdpa_kernel(
+            backends=[torch.nn.attention.SDPBackend.FLASH_ATTENTION, torch.nn.attention.SDPBackend.MATH]
+        ):
             out = self.attention(
```

Comment:
could be good to add some logging to show which of the two algorithms is running

Reply:
Thanks @cathalobrien , could we at least add a comment about the logic in the code and to explain those kernels as just available for the SPDA?